### PR TITLE
feature: add new api endpoint #183

### PR DESCRIPTION
**Pull request by csae8092**
* /archiv/keyword-author-network/?authors=1&authors=4

### CODE REVIEW

**Comment by csae8092** [comment]
@stefanprobst feel free to suggest different query-param names

File: archiv/endpoint_views.py [hunk 1]
```diff
@@ -140,9 +141,12 @@ def get_queryset(self, **kwargs):
         return self.filter.qs.distinct()
 
     def render_to_response(self, context, **kwargs):
-        author_ids = self.request.GET.getlist('authors', [])
-        author_qs = Autor.objects.filter(id__in=author_ids)
-        qs = self.get_queryset().distinct().order_by('id')
+        author_ids = self.request.GET.getlist("authors", [])
```
Comment (stefanprobst) [inline archiv/endpoint_views.py:144]:
```suggestion
        author_ids = self.request.GET.getlist("author", [])
```

**Comment by kaggl** [comment]
One question form me: Whats the difference between this and just using keyword-network?text__autor=x&text__auto=y?
The most important feature missing is that there is no way to know which author(s) actually used the keyword. Is there a way to implement this? Ideally, the query would return additional nodes with kind: 'author' and corresponding edges, showing which author used which keywords

**Comment by kaggl** [comment]
nevermind, was using the old syntax

**Pull request merged**